.travis.yml

```diff
@@ -1,31 +1,25 @@
 language: ruby
 rvm: 2.0.0
-sudo: required
-os: linux
 
-matrix:
-  include:
-    - os: osx
-      env: OSX=10.11
-      osx_image: xcode7.3
-      rvm: system
-  allow_failures:
-    - os: linux
+env:
+  global:
+    - PATH="~/.linuxbrew/bin:~/.linuxbrew/sbin:$PATH"
+    - HOMEBREW_DEVELOPER=1
+    - LINUXBREW=$HOME/.linuxbrew
```

Comment:
I'm not sure this will work as intended. $HOME is defined to be /home/linuxbrew in the `before_install` section. So this *might* expand to /home/travis/.linuxbrew or the like.

Moving `HOME=/home/linuxbrew`  to right above this line would probably do the trick.